cloudbuild.yaml

```diff
@@ -206,3 +206,17 @@ steps:
       stack exec -- rad-daemon-radicle &
 
       stack test :e2e
+
+  - id: "Package"
+    waitFor:
+    - "Integration tests"
+    name: $_BUILD_IMAGE
+    env: ['STACK_ROOT=/workspace/.stack']
+    entrypoint: 'bash'
+    args:
+    - "-c"
+    - |
+      set -euxo pipefail
+
+      VERSION=g$COMMIT_SHA TARGET=debian ./packaging/build-package.sh
```

Comment:
What's the significance of the `g` here?

Reply:
This was intended to signify a build from the **g**it repo. I have changed it to use a more explicit versioning scheme based on the date and the commit sha.